django_ilmoitin/models.py

```diff
@@ -62,4 +64,4 @@ class Meta:
         verbose_name_plural = _("notifications")
 
     def __str__(self):
-        return str(self.type)
+        return notifications.registry[self.type]
```

Comment:
Handling of once created and then disabled notifications probably needs some love in Ilmoitin in general, but I think it is not optimal to make such cause crash here. Maybe something like:

```
if 'self.type' in notifications.registry:
    return notifications.registry[self.type]
else:
    return '{} ({})'.format(self.type, _('disabled'))
```
?

Reply:
yes, reasonable suggestion 👍